prometheus-stats

```diff
@@ -124,6 +128,21 @@ pr_retries_sum {acc}
 pr_retries_count {acc}
 '''
 
+    # S3 space usage
+    space_usage = {}
+    for uri in stores.HYBRID_STORES:
+        url = urllib.parse.urlparse(uri)
+
+        if s3.is_key_present(url):
+            space_usage[uri] = sum(int(size) for size, in s3.parse_list(s3.list_bucket(url), "Size"))
+    if space_usage:
+        output += """
+# HELP s3_usage_bytes gauge of how much disk space an S3 store is using
```

Comment:
The "gauge of" is fairly redundant, but this is also entirely unimportant.

Reply:
I copied it from the "histogram" of example above